test/features/copy-in-copy-out.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Copy-in and Copy-out Support
+    In order to facilitate easier file transfer between the host and guest,
+    transient supports copy-in and copy-out support. Given a path on the host and an
+    absolute path on the VM, transient can:
+    - copy the host file or directory to the guest directory before starting the VM
+    - copy the guest file or directory to the host directory after stopping the VM
+
+  @skip-in-ci
```

Comment:
You might just be doing this for testing (if so please ignore), but before we merge we really need to make this only not run the tests in the docker part of CI, rather than _all_ CI. We should just be able to make a new tag and makefile target to do that.